python/cudf/cudf/core/column/string.py

```diff
@@ -710,6 +710,21 @@ def wrap(self, width, **kwargs):
         `drop_whitespace`, `break_long_words`, `break_on_hyphens`,
         `expand_tabsbool` are not yet supported and will raise a
         NotImplementedError if they are set to any value.
+
+        This method currently achieves behavior matching R’s
+        stringr library str_wrap function, the equivalent
+        pandas implementation can be obtained using the
+        following parameter setting:
+
+            expand_tabs = False
+
+            replace_whitespace = True
+
+            drop_whitespace = True
+
+            break_long_words = False
+
+            break_on_hyphens = False
```

Comment:
Are these the Pandas defaults? If so we should throw a warning loud and clear if someone tries to diverge from here. If these aren't the default we should throw a warning that we don't match default behavior.

Reply:
These are pandas optional params, our current behavior resembles that particular optional parameters setting. They are not defaults, still would you want me throw a warning ? or raise error if they pass the param and it diverges from that specific value?

Reply:
According to this: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.str.wrap.html the defaults are:
```
expand_tabs: True
replace_whitespace: True
drop_whitespace: True
break_long_words: True
break_on_hyphens: True
```

So given we're not matching the defaults we should throw a warning that indicate where our current defaults differ from Pandas defaults, as well as throwing exceptions if someone tries to explicitly set something that we don't support yet.

Reply:
@kkraus14, done.